test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -1,17 +1,18 @@
 (ns puppetlabs.trapperkeeper.services.authorization.authorization-service-test
   (:require
-   [clojure.test :refer :all]
-   [clojure.string :as str]
-   [puppetlabs.trapperkeeper.app :refer [get-service]]
-   [puppetlabs.trapperkeeper.authorization.testutils :refer :all]
-   [puppetlabs.trapperkeeper.services :refer [defservice]]
-   [puppetlabs.trapperkeeper.services.authorization.authorization-service
-    :refer [authorization-service]]
-   [puppetlabs.trapperkeeper.testutils.bootstrap :refer [with-app-with-config]]
-   [puppetlabs.trapperkeeper.testutils.logging :refer [with-test-logging]]
-   [ring.mock.request :as mock]
-   [ring.util.response :refer [response]]
-   [schema.test :as schema-test])
+    [clojure.test :refer :all]
```

Comment:
Mostly indentation changes.  :hand: :hand: :gun:  Also added the `puppetlabs.ssl-utils.core` ns reference.